src/modules/config/env/dev.json

```diff
@@ -19,5 +19,6 @@
   "SENTRY_DSN": "https://d2d3f81155656b53fe7746a0b69b6e0e@o4504361728212992.ingest.sentry.io/4505748385169408",
   "AUTH_URL": "/auth",
   "INVITE_REFERRER_URL": "https://decentraland.zone/invite",
-  "REFERRAL_SERVER_URL": "https://communities.decentraland.zone/v1"
+  "REFERRAL_SERVER_URL": "https://communities.decentraland.zone/v1",
+  "WEARABLE_PREVIEW_URL": "https://wearable-preview-9h4yvczat-decentraland1.vercel.app"
```

Comment:
Should the zone URL be a Vercel app? Can we have one deployed to zone?

Reply:
it should use the zone URL, at the moment we haven't deployed the WearblePreview, so that's why this Vercel app is here